sysdump/sysdump.go

```diff
@@ -808,7 +813,7 @@ func (c *Collector) Run() error {
 	}
 
 	if loggedStart {
-		c.logWarn("Please note that depending on your Cilium version and installation options, this may be expected")
+		c.logWarn("Please note that depending on your Cilium version and installation Options, this may be expected")
```

Comment:
Nit: I don't think we want to change `s/options/Options` here and in other error and log messages.